src/main.js

```diff
@@ -71,6 +71,12 @@ try {
   const spellCheckerLocale = SpellChecker.getSpellCheckerLocale(app.getLocale());
   config = settings.loadDefault(null, spellCheckerLocale);
   console.log('Failed to read or upgrade config.json', e);
+  if (!config.teams.length && config.defaultTeam) {
```

Comment:
@yuya-oc Should we consider renaming the term "team" to a "server" to reduce confusion? I searched through the GitHub repo and realized we refer to servers as teams in many places. This is probably because of how multi-team support worked before the team sidebar introduced a few months ago.

I could do a search and replace of the terms across our files after this PR and https://github.com/mattermost/desktop/pull/600 are merged.

Reply:
@jasonblais Yeah, we should do that. However, "team" is already used in `config.json`. So to avoid confusing, I don't think we need to do in v3.8.